### PR TITLE
1.1.4 hotfix

### PR DESCRIPTION
**Pull request by mariano54**

### CODE REVIEW

**Comment by r1235613** [comment]
It is now work at Ubuntu 20.04.
I fellow [install tutor](https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL) checkout to there.

File: chia/util/condition_tools.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def created_outputs_for_conditions_dict(
         # and don't just fail with asserts
         puzzle_hash, amount_bin = cvp.vars[0], cvp.vars[1]
         amount = int_from_bytes(amount_bin)
-        coin = Coin(input_coin_name, puzzle_hash, amount)
+        coin = Coin(input_coin_name, puzzle_hash, uint64(amount))
```
Comment (arvidn) [inline chia/util/condition_tools.py:105]: this is strictly not necessary here, since it doesn't check negative values

**Comment by r1235613** [comment]
Hi, I use this version.
When I try to restart node, get error.
```
chia_full_node: Traceback (most recent call last):
  File "/home/r1235613/chia-blockchain/venv/bin/chia_full_node", line 33, in <module>
    sys.exit(load_entry_point('chia-blockchain', 'console_scripts', 'chia_full_node')())
  File "/home/r1235613/chia-blockchain/chia/server/start_full_node.py", line 60, in main
    return run_service(**kwargs)
  File "/home/r1235613/chia-blockchain/chia/server/start_service.py", line 218, in run_service
    return asyncio.run(async_run_service(*args, **kwargs))
  File "/usr/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 603, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.8/selectors.py", line 468, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
  File "/home/r1235613/chia-blockchain/chia/server/start_service.py", line 168, in _accept_signal
    self.stop()
  File "/home/r1235613/chia-blockchain/chia/server/start_service.py", line 178, in stop
    self._node._close()
  File "/home/r1235613/chia-blockchain/chia/full_node/full_node.py", line 522, in _close
    if self.blockchain is not None:
AttributeError: 'FullNode' object has no attribute 'blockchain'
```

**Comment by Nikolaj-K** [comment]
@r1235613 From reading the code, I'd think this indicates that 
```python
class FullNode:
    # ...
    self.blockchain = await Blockchain.create(...)
    # ...
```
in the `FullNode`'s `_start` call didn't return.

I wonder if there's a coding scheme (in this dynamically typed setup) that reduces that necessity for all the manual checks against `None`. I've seen several PR's aiming at patching those contingent situations, as well a lot of `try` blocks with far too long bodies.

**Review by hoffmang9 — APPROVED** [review]

**Pull request merged**